### PR TITLE
[office-js-preview] (Outlook) Add note on supported sensitivity levels

### PR DESCRIPTION
**Pull request by samantharamon**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: None.
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -19188,6 +19204,12 @@ declare namespace Office {
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
          *
+         * **Important**: Outlook on Mac, Outlook on the web, and the new Outlook on Windows (preview) only support Normal and Private sensitivity levels.
+         *
+         * **Errors**:
+         *
+         * - `Unsupported API parameter`: The sensitivity level set on an appointment isn't supported.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:19211]:
```suggestion
         * - `Unsupported API parameter`: Setting the sensitivity level of an appointment isn't supported.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -19207,6 +19229,12 @@ declare namespace Office {
...
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:19236]:
```suggestion
         * - `Unsupported API parameter`: Setting the sensitivity level of an appointment isn't supported.
```

**Review by AlexJerabek — APPROVED** [review]
Approving with one suggestion.

**Review by navya9singh — APPROVED** [review]

**Comment by samantharamon** [comment]
Ready to merge

**Pull request merged**